prow/spyglass/lenses/junit/template.html

```diff
@@ -46,7 +46,7 @@
       <td class="mdl-data-table__cell--non-numeric" style="color: #FF0000">{{$test.Junit.Status}}</td>
       <td class="mdl-data-table__cell--non-numeric">{{$test.Junit.Duration}}</td>
       <td class="mdl-data-table__cell--non-numeric">{{$test.Junit.Error.Message}}</td>
-      <td class="mdl-data-table__cell--non-numeric">{{$test.Junit.Error.Body}}</td>
+      <td class="mdl-data-table__cell--non-numeric"></td>
```

Comment:
Why keep the empty cell?

Reply:
to match the other tables... which also have empty cells? :thinking:

Reply:
But this one didn't have this one until #9490! I think?

??????

???????????

/pony confused

Reply:
yeeeppppp no idea what I was doing

Reply:
I will keep only the needed amount of filler so that tables can be the same size.